guides-markdown/multisig-tx.md

```diff
@@ -4,6 +4,10 @@
 Nodar Chkuaselidze
 ```
 
+```post-description
+Working with transactions using bcoin and the MTX and TX objects.
```

Comment:
Post description should be something different for the multisig. Maybe: "Working with and creating multi-signature transactions using built in bcoin utilities and the wallet api."